spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/inbound/Mqttv5PahoMessageDrivenChannelAdapter.java

```diff
@@ -90,7 +90,7 @@ public class Mqttv5PahoMessageDrivenChannelAdapter extends AbstractMqttMessageDr
 	public Mqttv5PahoMessageDrivenChannelAdapter(String url, String clientId, String... topic) {
 		super(url, clientId, topic);
 		this.connectionOptions = new MqttConnectionOptions();
-		this.connectionOptions.setServerURIs(new String[]{ url });
+		this.connectionOptions.setServerURIs(new String[] {url});
```

Comment:
Please, revert such a change: our code style is exactly opposite of what you have done 😄 

We still have an opened issue to manage code style template via IDE from the project perspective: https://github.com/spring-projects/spring-integration/issues/3686.
Therefore meanwhile everything is manual. 
Sorry for inconvenience!